store/tikv/snapshot.go

```diff
@@ -404,6 +407,30 @@ func extractKeyErr(keyErr *pb.KeyError) error {
 	return errors.Errorf("unexpected KeyError: %s", keyErr.String())
 }
 
+func prettyLockNotFoundKey(rawRetry string) string {
+	if !strings.Contains(rawRetry, "TxnLockNotFound") {
```

Comment:
TxnLockNotFound is the error message from TiKV?

Reply:
yes, TiKV define a struct named `TxnLockNotFound` in rust..and will output message like `Txn(Mvcc(TxnLockNotFound { start_ts: 408090278408224772, commit_ts: 408090279311835140,`... :disappointed: